xmtp_mls/migrations/2023-10-29-205333_state_machine_init/up.sql

```diff
@@ -49,10 +49,10 @@ CREATE TABLE group_intents (
     -- INTENT_STATE,
     "state" INT NOT NULL,
     -- The hash of the encrypted, concrete, form of the message if it was published.
-    "payload_hash" BLOB,
+    "payload_hash" BLOB UNIQUE,
```

Comment:
This feels safe to be unique. We should never have more than one intent pointing to a single payload